### PR TITLE
7.0 pos backup draft orders

### PR DESCRIPTION
**Pull request by legalsylvain**
Propose 'pos_backup_draft_orders' in 7.0 serie.

This PR has been previously approved on launchpad:
- @pedrobaeza :+1:  2014-07-30

Ref: https://code.launchpad.net/~sylvain-legal/openerp-pos/7.0_pos_backup_draft_orders/+merge/228829

### CODE REVIEW

**Comment by legalsylvain** [comment]
Coverage 100%, Nice !

**Comment by pedrobaeza** [comment]
Indeed! Although JS is not covered in coveralls I'm afraid.

**Comment by max3903** [comment]
POT file is missing

**Comment by legalsylvain** [comment]
@pedrobaeza Yes. It was a joke. BTW do you know good and easy way to realize JS test on Odoo ? I didn't saw such test in Odoo Module but for web or pos addons it is necessary. Most bugs are from client side.

@max3903 : Thanks. Fixed.

File: pos_backup_draft_orders/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            """in Point Of Sale (Front end)""",
+    'version': '0.1',
+    'category': 'Point Of Sale',
+    'description': """
+Allow users to backup draft orders in Point Of Sale (Front end)
+===============================================================
+
+Functionality:
+--------------
+    * Allow user to backup in the backoffice side orders tiped in front-end;
```
Comment (max3903) [inline pos_backup_draft_orders/__openerp__.py]: I think you mean "ticked off" or "selected"

**Comment by pedrobaeza** [comment]
@legalsylvain There is the possibility of making JS tests in v7, and improve on v8 (https://www.odoo.com/apps/web_tests_demo/), but they are not covered yet on coveralls for maintainer quality tools.

**Comment by legalsylvain** [comment]
Thanks, i'll take a look on that in a while. Regards.

**Comment by legalsylvain** [comment]
@max3903 Is it now OK for you ?
Regards.

**Comment by sylvainc** [comment]
:+1:

**Pull request merged**